acceptance/tests/rototillerTask/command_arguments_with_override.rb

```diff
@@ -0,0 +1,78 @@
+require 'beaker/hosts'
+require 'rakefile_tools'
+
+test_name 'C97827: can set envvar to override command name when using task.command' do
+  extend Beaker::Hosts
+  extend RakefileTools
+
+  step 'Test command with an override_env that has a value' do
+
+    override_env = 'BROCKSAMSON'
```

Comment:
probably should use a different env here than the other test.
we might want to assert we aren't using an env that already exists in these tests